Website/openrails.org/shared/menu.php

```diff
@@ -34,7 +34,8 @@ function in_directory($cwd, $name) {
                 <!--<li><a href="/discover/version-1-1/">Version 1.1</a></li>-->
                 <!-- <li><a href="/discover/version-1-2/">Version 1.2</a></li> -->
                 <!-- <li><a href="/discover/version-1-3/">Version 1.3</a></li> -->
-                <li><a href="/discover/version-1-4/">Version 1.4</a></li>
+                <!-- <li><a href="/discover/version-1-4/">Version 1.4</a></li> -->
+                <li><a href="/discover/version-1-5-1/">Version 1.5.1</a></li>
```

Comment:
Put back to discover 1.5 page

Reply:
Changes made.

James, please approve again.
Assumubg it's OK, please re-run the script that populates openrails.org, so I can reconcile with Ryan's website repo.